hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -67,7 +67,7 @@ def lambda_handler(event, context):
     response_iterator = paginator.paginate(
```

Comment:
nit: do we need to paginate? dynamodb will happily return 1MB of data. Are we expecting the list of configs to be that long, or are we being defensive?

Reply:
yes this part will be changed using config framework in a follow-up pull request